modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,91 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
 
-<h2>Moving between UI sections</h2>
+<h1>Navigate between UI sections</h1>
 
-<p>When keyboard navigation is active, pressing tab will move the focus to the next major section of the UI, where applicable. These sections are:</p>
-<ul>
-  <li>the menubar</li>
-  <li>each group of the toolbar </li>
-  <li>the sidebar</li>
-  <li>the element path in the footer </li>
-  <li>the wordcount toggle button in the footer </li>
-  <li>the branding link in the footer </li>
-  <li>the editor resize handle in the footer</li>
-</ul>
+<p>To move from one UI section to the next, press <strong>Tab</strong>.</p>
+
+<p>To move from one UI section to the previous, press <strong>Shift+Tab</strong>.</p>
+
+<p>The <strong>Tab</strong> order of these UI sections is:
+
+<ol>
+<li>Menu bar</li>
+<li>Toolbar group 1</li>
+<li>Toolbar group n</li>
+<li>Sidebar</li>
+<li>Document footer</li>
+<li>Word count toggle button in the TinyMCE footer</li>
+<li>Branding link in the TinyMCE footer</li>
+<li>Editor resize handle in the TinyMCE footer</li>
+</ol>
 
-<p>Pressing shift + tab will move backwards through the same sections, except when moving from the footer to the toolbar. Focusing the element path then pressing shift + tab will move focus to the first toolbar group, not the last.</p>
+<p>If a UI section is not present, it is skipped.</p>
 
-<h2>Moving within UI sections</h2>
+<p>If the document footer has keyboard navigation focus, and there is no visible sidebar, pressing <strong>Shift+Tab</strong> moves focus to <strong>Toolbar group 1</strong>, not <strong>Toolbar group n</strong>.
+
+<h1>Navigate within UI sections</h1>
+
+<p>To move from one UI element to the next, press the appropriate <strong>Arrow</strong> key.</p>
+
+<p>The <strong>Left</strong> and <strong>Right</strong> arrow keys</p>
 
-<p>Keyboard navigation within UI sections can usually be achieved using the left and right arrow keys. This includes:</p>
 <ul>
-  <li>moving between menus in the menubar</li>
-  <li>moving between buttons in a toolbar group</li>
-  <li>moving between items in the element path</li>
+<li>move between menus in the menu bar.</li>
+<li>open a sub-menu in a menu.</li>
+<li>move between buttons in a toolbar group.</li>
+<li>move between items in a document footer’s element path.</li>
 </ul>
 
-<p>In all these UI sections, keyboard navigation will cycle within the section. For example, focusing the last button in a toolbar group then pressing right arrow will move focus to the first item in the same toolbar group. </p>
+<p>The <strong>Down</strong> and <strong>Up</strong> arrow keys
+
+<ul>
+<li>move between menu items in a menu.</li>
+<li>move between items in a toolbar pop-up menu.</li>
+</ul>
 
-<h1>Executing buttons</h1>
+<p><strong>Arrow</strong> keys cycle within the focussed UI section or element.</p>
```

Comment:
"focused" more common than "focussed" in US English.
```suggestion
<p><strong>Arrow</strong> keys cycle within the focused UI section.</p>
```